From 14d8bd430a720dd6a01d18440fafa015bdeb5260 Mon Sep 17 00:00:00 2001
From: Vincent ABRIOU <vincent.abriou@st.com>
Date: Fri, 29 Jan 2021 10:39:10 +0100
Subject: [PATCH 1/1] TFLite: tools: make: remove test_delegate_providers.cc
 from the build resources for libtensorflow-lite.a

lite/kernel/test_delegate_providers.cc should not be include in
libtensorflow-lite.a but it was not par of CORE_CC_EXCLUDE_SRCS because the
file syntax was not recognized as an excluded file.
This patch propose to exclude all the file containing *test*.cc.

Signed-off-by: Vincent ABRIOU <vincent.abriou@st.com>
---
 tensorflow/lite/tools/make/Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tensorflow/lite/tools/make/Makefile b/tensorflow/lite/tools/make/Makefile
index 9d90e9526be..617e818c87f 100644
--- a/tensorflow/lite/tools/make/Makefile
+++ b/tensorflow/lite/tools/make/Makefile
@@ -171,6 +171,7 @@ CORE_CC_EXCLUDE_SRCS := \
 $(wildcard tensorflow/lite/*test.cc) \
 $(wildcard tensorflow/lite/*/*test.c) \
 $(wildcard tensorflow/lite/*/*test.cc) \
+$(wildcard tensorflow/lite/*/*test*.cc) \
 $(wildcard tensorflow/lite/*/*/benchmark.cc) \
 $(wildcard tensorflow/lite/*/*/example*.cc) \
 $(wildcard tensorflow/lite/*/*/test*.cc) \
@@ -186,8 +187,6 @@ $(wildcard tensorflow/lite/*/*/*/*/*/example*.cc) \
 $(wildcard tensorflow/lite/*/*/*/*/*/test*.cc) \
 $(wildcard tensorflow/lite/*/*/*/*/*/*test.cc) \
 $(wildcard tensorflow/lite/*/*/*/*/*/*tool.cc) \
-$(wildcard tensorflow/lite/kernels/*test_main.cc) \
-$(wildcard tensorflow/lite/kernels/*test_util*.cc) \
 $(wildcard tensorflow/lite/tools/make/downloads/cpuinfo/src/*/mock*.c) \
 tensorflow/lite/tflite_with_xnnpack.cc \
 $(MINIMAL_SRCS)
-- 
2.17.1

